unix/xserver/hw/vnc/xvnc.c

```diff
@@ -111,7 +111,8 @@ static Bool vncPixmapDepths[33];
 static Bool Render = TRUE;
 
 static Bool displaySpecified = FALSE;
-static char displayNumStr[16];
+
+extern char displayStr[32];  /* This is where we pass the display to PAM modules */
```

Comment:
I think it's more modular to let them keep their own two copies of things.

Reply:
ACK